tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -432,47 +433,44 @@ def test_modulus(device, backend, random_state=42):
     """
     Tests the stability and differentiability of modulus
     """
-    torch.manual_seed(random_state)
-    # Test with a random vector
-    x = torch.randn(2, 4, 128, 2, requires_grad=True, device=device)
-
-    if backend.name == 'torch_skcuda' and device == 'cpu':
-        # If we are using a GPU-only backend, make sure it raises the proper
-        # errors for CPU tensors.
+    if backend.name == "torch_skcuda" and device == "cpu":
         with pytest.raises(TypeError) as re:
             x_bad = torch.randn((4, 2)).cpu()
-            backend.modulus_complex(x_bad)
+            backend.modulus(x_bad)
         assert "for CPU tensors" in re.value.args[0]
         return
 
+    torch.manual_seed(random_state)
+    # Test with a random vector
+    x = torch.randn(2, 4, 128, 2, requires_grad=True, device=device)
 
-    x_abs = backend.modulus_complex(x)
+    x_abs = backend.modulus(x).squeeze(-1)
+    assert len(x_abs.shape) == len(x.shape[:-1])
 
-    assert len(x_abs.shape) == len(x.shape)
     # check the value
     x_abs2 = x_abs.clone()
     x2 = x.clone()
-    assert torch.allclose(x_abs2[..., 0], torch.sqrt(x2[..., 0]**2 + x2[..., 1]**2))
+    assert torch.allclose(x_abs2, torch.sqrt(x2[..., 0] ** 2 + x2[..., 1] ** 2))
 
     with pytest.raises(TypeError) as te:
         x_bad = torch.randn(4).to(device)
-        backend.modulus_complex(x_bad)
+        backend.modulus(x_bad)
     assert "should be complex" in te.value.args[0]
 
     if backend.name == "torch_skcuda":
         pytest.skip("The skcuda backend does not pass differentiability"
-            "tests, but that's ok (for now).")
+                "tests, but that's ok (for now).")
```

Comment:
Why extra whitespace?

Reply:
thats just what the indentation is like.

Reply:
It wasn't before. Why are we changing it?